core/src/apps/management/apply_settings.py

```diff
@@ -12,6 +12,8 @@
 
 
 async def apply_settings(ctx: wire.Context, msg: ApplySettings):
+    if not storage.is_initialized():
+        raise wire.NotInitialized("Device is not initialized")
```

Comment:
Hmm, there are two fields that might be of interest here: the display rotation and maybe the label, if it shows up during Shamir recovery.

Reply:
You can still set the label in the RecoveryDevice message. We can add also the rotation there but I think it is so rarely used feature we may remove such use case.

Btw another solution is indeed to remove all the settings from Recovery and Reset messages and apply the settings using the ApplySettings message. I belive that is clearer solution but since ti requires changes in Suite we decided to go this way.